src/Manager/Indices.php

```diff
@@ -90,13 +93,46 @@ public function clearCurrentIndexLanguage() {
 		$this->currentIndexLanguage = '';
 	}
 
+	/**
+	 * @return string[]
+	 */
+	private function getClusterIndicesCache() {
+		if ( null === self::$clusterIndices ) {
+			$clusterIndices = $this->elasticsearch->get_cluster_indices();
+			self::$clusterIndices = wp_list_pluck( $clusterIndices, 'index' );
+		}
+
+		return self::$clusterIndices;
+	}
+
+	/**
+	 * @param string $indexName
+	 */
+	private function saveIndexInClusterCache( $indexName ) {
+		if ( null === self::$clusterIndices ) {
```

Comment:
`self::getClusterIndicesCache()` - shouldn't it "load cache" and add it there? I ask out of curiosity